tests/common/devices.py

```diff
@@ -193,3 +194,33 @@ def get_crm_resources(self):
                         "used_count": int(fields[2]), "available_count": int(fields[3])})
 
         return result
+
+    def get_pmon_daemon_list(self):
+        """
+        @summary: get pmon daemon list from the config file (/usr/share/sonic/device/{platform}/{hwsku}/pmon_daemon_control.json)
+                  if some daemon is disabled in the config file, then remove it from the daemon list.
+        """
+        full_daemon_tup = ('xcvrd', 'ledd', 'psud', 'syseepromd')
+        daemon_ctl_key_prefix = 'skip_'
+        daemon_list = []
+        daemon_config_file_path = os.path.join('/usr/share/sonic/device', self.facts["platform"], 'pmon_daemon_control.json')
+        
+        try:
+            output = self.shell('cat %s' % daemon_config_file_path)
+            json_data = json.loads(output["stdout"])
+            logging.debug("original file content is %s" % str(json_data))
```

Comment:
need to decide on same methodology in logging information, should it start with capital letter or not. 
if yes, let's make all in the same concept.

Reply:
agree, I would keep it unchanged currently, will do it when aligned.

Reply:
Please update the file with one methodology. No reason to push new PR with mixed terminology while new added lines are not stick to one.

Reply:
went through the files and they are starting with capital letter, now aligned with it in the newly added files.